src/queue/eventhub/utils.ts

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable sort-keys */
+import {
+  connect,
+  disconnectWithoutError
+} from "@pagopa/fp-ts-kafkajs/dist/lib/KafkaOperation";
+import {
+  AzureEventhubSas,
+  AzureEventhubSasFromString
+} from "@pagopa/fp-ts-kafkajs/dist/lib/KafkaProducerCompact";
+import { defaultLog } from "@pagopa/winston-ts";
+import * as E from "fp-ts/Either";
+import * as IO from "fp-ts/IO";
+import * as TE from "fp-ts/TaskEither";
+import { constVoid, pipe } from "fp-ts/lib/function";
```

Comment:
```suggestion
import { constVoid, pipe } from "fp-ts/function";
```
It seems that importing from /function instead of /lib/functions produces a smaller bundle size